.github/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+# <!-- Use the title to describe PR changes in the imperative mood --> #
+  <!-- Remember this title will end up as the merge commit subject -->
+
+## 🗣 Description ##
+
+<!-- Describe the "what" of your changes in detail. -->
+<!-- To avoid scope creep, limit changes to a single goal. -->
+
+## 💭 Motivation and context ##
+
+<!-- Why is this change required? -->
+<!-- What problem does this change solve? How did you solve it? -->
+<!-- Mention any related issue(s) here using appropriate keywords such -->
+<!-- as "closes" or "resolves" to auto-close them on merge. -->
+
+## 🧪 Testing ##
+
+<!-- How did you test your changes? How could someone else test this PR? -->
+<!-- Include details of your testing environment, and the tests you ran to -->
+<!-- see how your change affects other areas of the code, etc. -->
+
+<!--
+## 📷 Screenshots (if appropriate) ##
+
+Uncomment this section if a screenshot is needed.
+
+-->
+
+## ✅ Pre-approval checklist ##
+
+<!-- Remove any of the following that do not apply. -->
+<!-- Draft PRs should have one or more unchecked boxes. -->
+<!-- If you're unsure about any of these, don't hesitate to ask. -->
+<!-- We're here to help! -->
+
+- [ ] This PR has an informative and human-readable title
+- [ ] PR is targeting the correct parent branch for merge
+- [ ] Changes are limited to a single goal - *eschew scope creep!*
+- [ ] *All* future TODOs are captured in issues, which are referenced
+      in code comments
+- [ ] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
+- [ ] All relevant type-of-change labels have been added
+- [ ] All relevant project fields have been set
+- [ ] All relevant repo and/or project documentation has been updated
+      to reflect the changes in this PR
+- [ ] Tests have been added and/or modified to cover the changes in this PR
```

Comment:
Suggest we spell out the types of tests that are expected, here or in comments in the testing section.  I'm thinking:

1) Make sure unit test coverage (with appropriate mocks) is added to cover powershell and rego changes
2) Make sure functional tests are added/updated for major changes to baseline content
3) In addition, ensure smoke tests continue to pass.